### PR TITLE
Fix for issue #6

### PR DESCRIPTION
**Pull request by EdwinVW**
Added fix for issue - I added code to save the current directory before it is changed and reset is afterwards.

### CODE REVIEW

File: src/Augurk.CommandLine/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -46,6 +46,6 @@
 //
 // You can specify all the values or you can default the Revision and Build Numbers 
 // by using the '*' as shown below:
-[assembly: AssemblyVersion("3.0.0")]
-[assembly: AssemblyFileVersion("3.0.0")]
-[assembly: AssemblyInformationalVersion("3.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("3.0.1")]
```
Comment (jmezach) [inline src/Augurk.CommandLine/Properties/AssemblyInfo.cs]: Please undo the changes to this file (versioning is handled through GitVersion.yml)

File: src/Augurk.CommandLine/Commands/PublishCommand.cs [hunk 2]
```diff
@@ -252,6 +252,7 @@ private Feature ParseFeatureFile(string featureFile)
         {
             using (var reader = new StreamReader(featureFile))
             {
+                Stack<string> directories = new Stack<string>();
```
Comment (jmezach) [inline src/Augurk.CommandLine/Commands/PublishCommand.cs]: Since we're only looping over one file at the same time, wouldn't simply storing the current directory in a local variable suffice, rather than creating a Stack which will only ever contain one item?

**Review by jmezach — CHANGES_REQUESTED** [review]
Please have a look at the posted comments

File: src/Augurk.CommandLine/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (EdwinVW) [reply src/Augurk.CommandLine/Properties/AssemblyInfo.cs]: Undo, or change back to 3.0.0?

File: src/Augurk.CommandLine/Commands/PublishCommand.cs (hunk 2 above)
Comment (EdwinVW) [reply src/Augurk.CommandLine/Commands/PublishCommand.cs]: Check. I created the stack because I anticipated more directory-changes. Will change this.

File: src/Augurk.CommandLine/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (jmezach) [reply src/Augurk.CommandLine/Properties/AssemblyInfo.cs]: ;) Either. It's all the same anyway.

File: src/Augurk.CommandLine/Commands/PublishCommand.cs (hunk 2 above)
Comment (jmezach) [reply src/Augurk.CommandLine/Commands/PublishCommand.cs]: Thanks!

**Review by jmezach — APPROVED** [review]

**Comment by jmezach** [comment]
@EdwinVW Would you mind updating the release notes in the .nuspec file?

**Comment by EdwinVW** [comment]
Changed the release-notes. Not entirely sure about the format. Please review.

**Comment by jmezach** [comment]
@EdwinVW They're fine :).

We've just published version 3.1.1 to NuGet. Might take some time before it shows up in the feed though. Thanks for the fix!
